webui/src/models/cluster/zones/index.ts

```diff
@@ -0,0 +1,39 @@
+import { combine, guard, restore } from 'effector';
+
+import { Maybe, app } from 'src/models';
+
+const { not, some } = app.utils;
+
+// events
+export const setServerZoneEvent = app.domain.createEvent<{ uuid: string; zone?: string }>('set server sone click');
+export const zoneAddModalOpenEvent = app.domain.createEvent<{ uuid: string }>('zone adding modal open');
+export const zoneAddModalCloseEvent = app.domain.createEvent('zone adding modal close');
+export const zoneAddModalSetValueEvent = app.domain.createEvent<string>('zone adding modal set value');
+export const zoneAddModalSubmitEvent = app.domain.createEvent('add server zone modal submit');
+
+// stores
+export const $zoneAddModalError = app.domain.createStore<string>('');
+export const $zoneAddModalValue = restore(zoneAddModalSetValueEvent, '');
```

Comment:
Is it specially created without a domain?

Reply:
Thanks, fixed.